portal-api/src/v1/controllers/utilisation-report-service/utilisation-report-upload.controller.test.js

```diff
@@ -0,0 +1,45 @@
+const { when } = require('jest-when');
```

Comment:
I've recently fixed the api tests for this function, which now do a happy path through the utilisation-report-upload.controller. I'm not sure what we do with the other uploading to azure type functions and if the api-tests just upload to azure but we should try copy that so they work. 
Grab me offline if you want to chat about this